app/next-client-app/next.config.js

```diff
@@ -3,7 +3,7 @@ const nextConfig = {
   output: "standalone",
   experimental: {
     serverActions: {
-      allowedOrigins: [process.env.BACKEND_URL],
+      allowedOrigins: [process.env.BACKEND_ORIGIN],
```

Comment:
Do leave a comment as to what works for this value in local development - and I agree, that having it distinct from `BACKEND_URL` is helpful, as it might need to be set to something different when deployed.

Reply:
As discussed - keep you original change here to `BACKEND_ORIGIN`, but just leave a comment here as to what works locally for you, and should be in our `.env` files.